packages/restitution/src/HeaderRestitution.tsx

```diff
@@ -0,0 +1,65 @@
+import  * as React from 'react';
+import classNames from 'classnames';
+import {
+  PropsManager,
+  withClassDefault,
+  withClassModifier,
+  WithClassModifierOptions,
+} from '@axa-fr/react-toolkit-core';
+import { compose } from 'recompose';
+
+const DEFAULT_CLASSNAME = 'af-restitution__header';
+
+type HeaderRestitutionBaseProps = {
+  className?: string;
+  children?: React.ReactNode;
+} 
+
+const defaultProps: Partial<HeaderRestitutionBaseProps> = {
+  children: null,
+  className:DEFAULT_CLASSNAME
+};
+
+interface HeaderRestitutionProps extends WithClassModifierOptions, HeaderRestitutionBaseProps {}
+
+const HeaderRestitution: React.SFC<HeaderRestitutionProps> = ({ children, title, subtitle, titleLeft, className }: { children:React.ReactNode, title:React.ReactNode, subtitle:React.ReactNode, titleLeft:React.ReactNode, className:string }) => {
+  return (
+    <header className={className}>
+      <div className="af-restitution__header-left">
+        <div className="af-restitution__title">
+          {title && <span className="af-restitution__title-title">{title}</span>}
+          <div className="af-restitution__title-subtitle">{subtitle}</div>
```

Comment:
Should use a more semantic HTML Tag here, a content tag like Hx or span

Reply:
agree but it is copy paste from 
https://axaguildev.github.io/react-toolkit/latest/design/index.html#/restitution
we should change both
What do you think about h3?

Reply:
yes we should change both, ok for 'h3'